app/src/main/java/my/noveldokusha/scraper/sources/ReadWn.kt

```diff
@@ -0,0 +1,103 @@
+package my.noveldokusha.scraper.sources
+
+import com.google.gson.JsonParser
+import my.noveldokusha.data.BookMetadata
+import my.noveldokusha.data.ChapterMetadata
+import my.noveldokusha.scraper.*
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+
+
+class ReadWn : SourceInterface.catalog
```

Comment:
Wrong PR? This is not NovelHall :)

Reply:
There seems to be error will delete and recreate the pr